tests/test_device.py

```diff
@@ -1154,3 +1154,50 @@ async def test_somrig_events(zha_gateway: Gateway) -> None:
             )
         )
     ]
+
+
+async def test_symfonisk_events(
+    zha_gateway: Gateway,
+) -> None:
+    """Test that Symfonisk events are handled correctly."""
+
+    zigpy_dev = await zigpy_device_from_json(
+        zha_gateway.application_controller,
+        "tests/data/devices/ikea-of-sweden-symfonisk-sound-remote-gen2.json",
+    )
+
+    zha_device = await join_zigpy_device(zha_gateway, zigpy_dev)
+
+    listener = mock.Mock()
+    zha_device.on_all_events(listener)
+
+    # ShortcutV1Cluster:shortcut_v1(shortcut_button=1, shortcut_event=1)
```

Comment:
nit: I think the comment should be renamed back to the actual cmd name `shortcut_v1_events`?
```suggestion
    # ShortcutV1Cluster:shortcut_v1_events(shortcut_button=1, shortcut_event=1)
```